Neos.Neos/Classes/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -271,6 +271,18 @@ protected function setGlobalUiElementLabels($nodeTypeName, array &$configuration
                 }
             }
         }
+
+        $creationDialogConfiguration = Arrays::getValueByPath($configuration, 'ui.creationDialog.elements');
+        if (is_array($creationDialogConfiguration)) {
+            foreach ($creationDialogConfiguration as $elementName => $elementConfiguration) {
+                if (!is_array($elementConfiguration) || !$this->shouldFetchTranslation($elementConfiguration['ui'])) {
+                    continue;
+                }
+                $translationLabelId = $this->getCreationDialogElementTranslationId($nodeTypeLabelIdPrefix,
```

Comment:
this method call seems a bit overkill when it could just be `$translationLabelId = $nodeTypeSpecificPrefix . 'creationDialog.' . $elementName;` right here instead

Reply:
The idea was to use the same style for all, `getPropertyLabelTranslationId` is also only used once. But I will push a version including your change request.

Reply:
I see, you could use the existing `getInspectorElementTranslationId` method instead then right?

    $translationLabelId = $this->getInspectorElementTranslationId($nodeTypeLabelIdPrefix, 'creationDialog', $elementName);

that would make it more consistent, although also longer

Reply:
Ah, good catch! I pushed an improved version.